### ISSUE TITLE
Path of imported media depend on place whence *Olive* binary launched

### ISSUE THREAD

**Issue by Symbian9** [olive-editor/olive#448]
Seems like path stored in project files depend on place whence *Olive* binary launched.

Need review path recognition and related path writing inside `.ove` files.

**Comment by itsmattkc** [comment]
The path stored is always relative to where the project saved, unless the AppImage messes this up somehow.

**Comment by Symbian9** [comment]
Maybe, this is not just AppImage related issue, but Linux specific (@probonopd ?)

Open terminal in other than user's root folder (`me@linux:~`)

```
me@linux:~/AppImages
$ ./Olive-*.AppImage /home/me/Videos/test.ove
[INFO] Failed to match "Videos/screencast.mp4" to file ((null):0, (null))
```

Both files, `test.ove` & `screencast.mp4`, stored in `/home/me/Videos` (aka `~/Videos`).

If I open terminal from `/home/me` (aka `~`) and execute `$ /home/me/AppImages/Olive-*.AppImage /home/me/Videos/test.ove` than imported file shown in project as normally.

**Comment by itsmattkc** [comment]
Could you send the `test.ove` file? It should have more information about this issue.

**Comment by Symbian9** [comment]
> Could you send the `test.ove` file?

Here it is:

```
<?xml version="1.0" encoding="UTF-8"?>
<project>
    <version>190201</version>
    <url>/home/me/Videos/test.ove</url>
    <folders/>
    <media>
        <footage id="1" folder="0" name="screencast.mp4" url="Videos/screencast.mp4" duration="83870467" using_inout="0" in="0" out="0" speed="1" alphapremul="0" proxy="0" proxypath="">
            <video id="0" width="1280" height="720" framerate="29.9700299700" infinite="0"/>
            <audio id="1" channels="1" layout="4" frequency="44100"/>
        </footage>
    </media>
    <sequences>
        <sequence id="1" folder="0" name="Sequence 01" width="1280" height="720" framerate="29.9700299700" afreq="44100" alayout="3" open="1" workarea="1" workareaIn="0" workareaOut="2513">
            <clip id="0" enabled="1" name="screencast.mp4" clipin="0" in="0" out="2513" track="0" opening="-1" closing="-1" r="128" g="128" b="192" autoscale="0" speed="1.0000000000" maintainpitch="0" reverse="0" type="0" media="1" stream="1">
                <linked>
                    <link id="1"/>
                </linked>
                <effect name="/Volume" enabled="1">
                    <row>
                        <field id="volume" value="100"/>
                    </row>
                </effect>
                <effect name="/Pan" enabled="1">
                    <row>
                        <field id="pan" value="0"/>
                    </row>
                </effect>
            </clip>
            <clip id="1" enabled="1" name="screencast.mp4" clipin="0" in="0" out="2513" track="-1" opening="-1" closing="-1" r="128" g="128" b="192" autoscale="0" speed="1.0000000000" maintainpitch="0" reverse="0" type="0" media="1" stream="0">
                <linked>
                    <link id="0"/>
                </linked>
                <effect name="Distort/Transform" enabled="1">
                    <row>
                        <field id="posx" value="640"/>
                        <field id="posy" value="360"/>
                    </row>
                    <row>
                        <field id="scalex" value="100"/>
                        <field id="scaley" value="100"/>
                    </row>
                    <row>
                        <field id="uniformscale" value="1"/>
                    </row>
                    <row>
                        <field id="rotation" value="0"/>
                    </row>
                    <row>
                        <field id="anchorx" value="0"/>
                        <field id="anchory" value="0"/>
                    </row>
                    <row>
                        <field id="opacity" value="100"/>
                    </row>
                    <row>
                        <field id="blendmode" value="0"/>
                    </row>
                </effect>
            </clip>
        </sequence>
    </sequences>
</project>
```

**Comment by itsmattkc** [comment]
Should be fixed as of 559509cdb01db44a20e83377c6cf616775fe0b33
